# Online/Offline
mapping_online: false
real_time: true

# Topics
points_topic: "/velodyne_points"
imus_topic: "/vectornav/IMU"

# Extrinsics
estimate_extrinsics: false
print_extrinsics: false
initial_gravity: [0.0, 0.0, -9.807]
I_Translation_L: [1.25, 0.0, 0.0]
I_Rotation_L: [
     1.0,  0.0,  0.0,
     0.0, -1.0,  0.0,
     0.0,  0.0, -1.0
]

# Delays
LiDAR_noise: 0.001
empty_lidar_time: 0.1      # Should be at least [FULL_ROTATION_TIME]
real_time_delay: 0.1       # Should be at least [FULL_ROTATION_TIME] (without a modificated LiDAR driver)

# LiDAR
full_rotation_time: 0.1
min_dist: 4
ds_rate: 4

# IMU
imu_rate: 400
covariance_gyroscope: 6.01e-4
covariance_acceleration: 1.53e-2
covariance_bias_gyroscope: 1.54e-5
covariance_bias_acceleration: 3.38e-4

# Localizator
MAX_NUM_ITERS: 3
# LIMITS: [0.001] * 23
NUM_MATCH_POINTS: 5
MAX_DIST_PLANE: 2.0

# Delta refinement
Heuristic:
    times: [0.5, 1.0, 1.25]
    deltas: [0.1, 0.05, 0.025, 0.01]